generator/Makefile

```diff
@@ -59,6 +59,7 @@ UBNT_AIROS_URL    := https://dl.ubnt.com/firmwares/airos-ubnt-mib/ubnt-mib.zip
 UBNT_AIROS_OLD_URL:= https://raw.githubusercontent.com/pgmillon/observium/refs/heads/master/mibs/FROGFOOT-RESOURCES-MIB
 UBNT_AIRFIBER_URL := https://dl.ubnt.com/firmwares/airfiber5X/v4.1.0/UBNT-MIB.txt
 UBNT_DL_URL       := http://dl.ubnt-ut.com/snmp
+UPS_MIB_URL       := https://raw.githubusercontent.com/pgmillon/observium/refs/heads/master/mibs/UPS-MIB
```

Comment:
Hmm, I wish there was a less indirect source for this MIB. I understand the authoritative definition is in [RFC 1628](https://datatracker.ietf.org/doc/html/rfc1628).

The closest thing I know about is the scripts in [net-snmp](https://github.com/net-snmp/net-snmp/tree/master/mibs) that can extract the data from RFCs.

Reply:
I could not find it in net-snmp's mibs.
that was the only place I could find a mib. if there is an alternative, happy to adjust the url.